src/File.php

```diff
@@ -1322,16 +1322,14 @@ public function forTemplate(): string
 
     /**
      * Return a html5 tag of the appropriate for this file (normally img or a)
-     *
-     * @return string
      */
-    public function getTag()
+    public function getTag(): string
     {
         $template = $this->File->getFrontendTemplate();
         if (empty($template)) {
             return '';
         }
-        return (string)$this->renderWith($template);
+        return $this->renderWith($template);
     }
```

Comment:
Just adding a bit of type safety since this method gets used in `forTemplate()` above.